spec/integration/session_spec.rb

```diff
@@ -1013,5 +1013,46 @@
         expect(@session.driver.send(:session_wait_time)).to eq 2
       end
     end
+
+    unless Capybara::Poltergeist.windows?
+      require 'pty'
+      require 'timeout'
+
+      context 'with pty' do
+        before do
+          Tempfile.open(%w(test rb)) do |file|
+            file.print(script)
+            file.flush
+
+            Timeout.timeout(10) do
+              PTY.spawn("bundle exec ruby #{file.path}") do |read, write, pid|
+                sleep 0.1 until read.readline.chomp == 'Please type enter'
```

Comment:
I found we'll get `"You're running an old version of PhantomJS, update to >= 2.1.1 for a better experience."` on travis

https://travis-ci.org/teampoltergeist/poltergeist/jobs/314940289#L719

Reply:
Yeah -- depending on which version PHANTOMJS  env var points to.  We test with multiple versions.

Reply:
It's ok if it's intentional 👍 
I left the comment just in case.

Reply:
Is the check for this string even necessary?

Reply:
Without this check, somehow  hangs and never dies 🤔 
With this hack, the spawned ruby process exits with non zero status code.

It's unnecessary if it's ok.

Reply:
Lets leave it there as long as the tests are passing (and failed without the stdin modification obviously)